### ISSUE TITLE
Regression: User models aren't displayed correctly anymore in Habbo.swf

### ISSUE THREAD

**Issue by stz33a** [ruffle-rs/ruffle#14007]
### Describe the bug

The walking animation of users doesn't play anymore, and the left arm isn't displayed correctly. This is happening since nightly 2023-11-13

I'm getting this error now:
```
ruffle_web-wasm_extensions.js:753 ERROR core/src/avm2/events.rs:424 Error dispatching event EventObject(EventObject { type: "DOWNLOAD_SET_DONE", class: flash.events::Event, ptr: 0x1895b940 }) to handler FunctionObject(FunctionObject { ptr: 0x16c8cc60 }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: appendToFigure)
```

![image](https://github.com/ruffle-rs/ruffle/assets/45462783/fd904a6a-c2fd-4411-961b-13fa75dc7cc6) ![walk_bad](https://github.com/ruffle-rs/ruffle/assets/45462783/1da03b9e-b24c-4e2d-881b-0a75c8efaa4a)

### Expected behavior

This is what it looks like in Flash:

![image](https://github.com/ruffle-rs/ruffle/assets/45462783/772ca46d-053c-4b8d-aa78-021e6f698dbd) ![walk_good](https://github.com/ruffle-rs/ruffle/assets/45462783/e3a479e3-dd04-4fb4-9364-bef051b7258c)

### Content Location

https://habbolegacy.com/flash_client

Here's the files from the base location: [r38.zip](https://github.com/ruffle-rs/ruffle/files/13310204/r38.zip)
And here's the deobfuscated Habbo.swf: [Link](https://cdn.discordapp.com/attachments/528241739006345239/1091341255373500496/RELEASE39-22643-22891-200911100035.tar.gz?ex=656099bf&is=654e24bf&hm=567d812c68e5ab56c4dea4e790f9cdeaed7444a08da3dc2264b82eb5d894c004&)

### Affected platform

Self-hosted version

### Operating system

Windows 10

### Browser

Google Chrome 119

**Comment by Lord-McSweeney** [comment]
Interesting, this didn't work for me the first time I tried it, but the second time, it worked fine:
[Habbo.webm](https://github.com/ruffle-rs/ruffle/assets/84632019/95ef902d-687d-415c-ab29-98e5796a8687)

**Comment by Lord-McSweeney** [comment]
Just to clarify- was it working in nightly 11-13, or was nightly 11-13 the first nightly with the bug?

**Comment by stz33a** [comment]
Nightly 11-13 was the first one with the bug

**Comment by Lord-McSweeney** [comment]
The only possible offending commit is 1501a858f6b8dea6a3ecb858dc5582d7a76debb4, then. @Aaron1011 can you check this?

**Comment by Aaron1011** [comment]
I think we need to implement the 'default namespace' logic for non-qname Multinames, which in this case will end up appending the public namespace to the incoming Multiname: https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/Toplevel.cpp#L287

**Comment by n0samu** [comment]
Fixed by #14801
